grails-app/controllers/org/pih/warehouse/reporting/ReportController.groovy

```diff
@@ -279,7 +279,7 @@ class ReportController {
         command.location = Location.get(session.warehouse.id)
         command.rootCategory = productService.getRootCategory()
 
-        def triggers = quartzScheduler.getTriggersOfJob(new JobKey("org.pih.warehouse.jobs.RefreshTransactionFactJob"))
+        def triggers = quartzScheduler.getTriggersOfJob(new JobKey("org.pih.warehouse.jobs.RefreshTransactionFactJob", "GRAILS_JOBS"))
```

Comment:
Where does this magic string come from?

Reply:
From quartzScheduler/jobStore/jobsByKey in the debugger 😀
![image](https://github.com/openboxes/openboxes/assets/83239466/15801ea0-93f1-4f83-b342-0421b70e5522)

Reply:
Sorry I meant where is it defined? In our code or the plugin?

Quick search on GitHub ... 
https://github.com/search?q=GRAILS_JOBS+repo%3Agrails-plugins%2Fgrails-quartz&type=code

reveals that it's defined as a constant in the Quartz plugin
```
GrailsJobClassConstants.DEFAULT_GROUP = "GRAILS_JOBS"
```
So we should use that. 

Or better yet, see if the getTriggers() method (that is dynamically added to the Job class by the plugin) returns the triggers.

    def triggers = RefreshTransactionFactJob.getTriggers()

Reply:
Looks like this 
```
def triggers = RefreshTransactionFactJob.getTriggers()
```
returns the closure from the job class so that's not what we want. 

So just clean up the magic string and call it a day.
```
def triggers = quartzScheduler.getTriggersOfJob(new JobKey("org.pih.warehouse.jobs.RefreshTransactionFactJob", GrailsJobClassConstants.DEFAULT_GROUP))
```